interface-impl-amf/src/main/java/org/mule/amf/impl/URIUtils.java

```diff
@@ -24,7 +28,12 @@ public static URI getPathAsUri(ApiReference apiRef) {
     }
   }
 
-  private static URI getUriFromFile(ApiReference apiRef) {
+  static String uriToPath(final URI uri) {
```

Comment:
This is used once , and is only one line, URLDecoder.decode(uri.toString())

Reply:
Updated